Dockerfile

```diff
@@ -32,12 +29,22 @@ RUN curl -L ${WOW_REPO}/archive/${WOW_REV}.zip > wow.zip \
   && rm wow.zip \
   && mv who-owns-what-${WOW_REV} who-owns-what
 
-COPY . /app
+ENV PYTHONUNBUFFERED yup
 
+# Note that these won't actually work until we either mount /app as a
+# volume or copy it over. For dev, this will be done via volume mount
+# by docker-compose; for prod the directory contents will be copied over
+# in that stage.
 WORKDIR /app
-
 CMD ["python", "load_dataset.py"]
 
-ENV PATH /var/pydev/bin:$PATH
-ENV PYTHONPATH /var/pydev
```

Comment:
I think that this is leftover from some earlier revision of our code where we installed stuff to `/var/pydev` or something.  I did a grep of the codebase and couldn't find anything that used this directory so I removed it, and things still seem to work okay.